packages/application-components/src/components/modal-pages/internals/modal-page-top-bar.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import { injectIntl } from 'react-intl';
+import {
+  Text,
+  CloseIcon,
+  FlatButton,
+  AngleLeftIcon,
+  SecondaryIconButton,
+  customProperties,
+} from '@commercetools-frontend/ui-kit';
+import messages from './messages';
+
+// Component to have a larger the clickable surface
+const LargeCloseIcon = props => (
+  <span
+    css={css`
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      &::after {
+        content: '';
+        position: absolute;
+        height: 35px;
+        width: 48px;
+        top: 0;
+        right: 0;
+      }
+    `}
+  >
+    <CloseIcon size="medium" {...props} />
+  </span>
+);
+LargeCloseIcon.displayName = 'LargeCloseIcon';
+
+const ModalPageTopBar = props => (
+  <div
+    css={css`
+      position: relative;
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      padding: ${customProperties.spacingS} ${customProperties.spacingM};
+      background-color: ${props.color === 'neutral'
+        ? customProperties.colorNeutral95
+        : customProperties.colorSurface};
+      border-bottom: 1px solid
+        ${props.color === 'neutral'
+          ? customProperties.colorSurface
+          : customProperties.colorNeutral};
+      & * + * {
+        margin-left: ${customProperties.spacingS};
+      }
+
+      /* FIXME: these "dirty" styles should be removed when the new Breadcrumbs component is implemented */
```

Comment:
Is there a task for that already?

Reply:
There isn't a task as in a "jira ticket", but design said it's planned as an upcoming new feature/component.